src/AbpHelper.Core/Templates/Crud/Groups/Repository/src/{{ProjectInfo.FullName}}.Domain/{{EntityInfo.RelativeDirectory}}/I{{EntityInfo.Name}}Repository.cs

```diff
@@ -6,11 +6,20 @@
     end
 ~}}
 using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
 using Volo.Abp.Domain.Repositories;
 
 namespace {{ EntityInfo.Namespace }}
 {
     public interface I{{ EntityInfo.Name }}Repository : {{ repository }}
     {
-    }
+        Task<{{ EntityInfo.Name }}Result> GetListAsync(
```

Comment:
What if there are two aggregate roots named `Action` and `ActionResult`?

Can we use `Task<(List<{{ EntityInfo.Name }}>, long)>` or `Task<GetEntityListResult<{{ EntityInfo.Name }}>>`?

Reply:
that would be better